### PR TITLE
Reference books

### PR DESCRIPTION
**Pull request by tenmillimaster**
#### Summary
```SUMMARY: Content "Adds reference books"```

#### Purpose of change
Some recipes would require information that likely cannot be memorized and wouldn't be located in common textbooks. 

#### Describe the solution
I've added a couple of works that I don't think one would really be able to learn from, but would be invaluable in specific recipes.

Want to know what solvent to attempt re-crystallization with? Use the CRC/merck manual.

Want to know whether your patient should/shouldn't take a certain drug with another drug or something? Use the PDR.

Want to know whether you can get decent welds with an arc welder using x base metal and y filler rod? Use the Machinery handbook.

Want to know how to use a certian IC circuit  you just pulled out of radio? Why, you'd want to use the datasheet archives. (Thanks whoever scraped them from the internet!)

So on and so forth.

### CODE REVIEW

**Comment by Xenomorph-III** [comment]
Will these be required in recipes soon enough?

**Comment by tenmillimaster** [comment]
I have some stuff planned, but it depends on when I get around to it

File: data/json/items/book/firstaid.json [hunk 1]
```diff
@@ -106,5 +106,39 @@
...
+    "intelligence": 4,
+    "time": "30 m",
+    "fun": -2
+  },
+  {
+    "id": "reference_firstaid2",
+    "type": "BOOK",
+    "name": "Merck Veterinary Manual",
+    "name_plural": "copies of The Merck Veterinary Manual",
+    "description": "This expansive text convers 'all domestic species and diseases in veterinary medicine worldwide'.  Should you need information regarding zoonotic diseases, off-label use of medication for animals, common dysfunctions and diseases of animals, or any other veterinary reference info, this is your book.  There are some interesting new chapters on animal cloning in this edition.",
```
Comment (Night-Pryanik) [inline data/json/items/book/firstaid.json]:
```suggestion
    "description": "This expansive text covers 'all domestic species and diseases in veterinary medicine worldwide'.  Should you need information regarding zoonotic diseases, off-label use of medication for animals, common dysfunctions and diseases of animals, or any other veterinary reference info, this is your book.  There are some interesting new chapters on animal cloning in this edition.",
```

**Comment by Rivet-the-Zombie** [comment]
So these don't actually *do* anything?

**Comment by vhzod** [comment]
Wouldn't most of the existing books have at least some reference information in them already, for example say the chemistry textbook

**Comment by tenmillimaster** [comment]
@rivet

Not yet. This is for higher order crafting. If you'd like to do a double displacement reaction, you're going to need to know relative solubilities: chemistry textbooks simply are not large enough.

if you'd like to confirm a melting point for purity, you will need a Merck handbook.

If you want to know which solvents are appropriate for your liquid liquid extraction of a chemical from a common store bought product, you will need this book.

If you're distilling something much more unique than ethanol, like, say, fractionating gasoline into it's subcomponents, you will want to know the boiling point of each constituent.

These books sole purpose is to replace the information that used to be contained on the internet.

**Comment by Rivet-the-Zombie** [comment]
@tenmillimaster, those are great ideas! I'd just kinda rather they had been (at least partially) implemented before this PR was complete.

**Comment by tenmillimaster** [comment]
> @tenmillimaster, those are great ideas! I'd just kinda rather they had been (at least partially) implemented before this PR was complete.
If I try to do them as 1 PR, I'll get far too OCD about it, get burnt out on expanding scope creep, and take a break again for a while, which I don't want.

**Comment by l29ah** [comment]
OCD got serious it seems.

**Pull request merged**